securepy/stdcapture.py

```diff
@@ -97,11 +101,11 @@ def stderr(self) -> str:
 
     def __enter__(self) -> None:
         """
-        Temporarely override `sys.stdout` and `sys.stderr`
+        Temporarely override `sys.stdout`, `sys.stdin` and `sys.stderr`
         to use `StringIO` to capture standard output & error.
```

Comment:
This is actually my mistake but it should say `LimitedStringIO`, since this PR is editing most of this file, it can be fixed here.
```suggestion
        to use `LimitedStringIO` to capture standard output & error.
```